apps/trustlab/src/payload/access/abilities.js

```diff
@@ -0,0 +1,60 @@
+import { checkRole } from "./checkRole";
+import { ROLE_ADMIN, ROLE_AUTHOR, ROLE_EDITOR } from "./roles";
+
+export const canManageContent = (user) => {
+  // Admin and editors can manage any content
+  if (checkRole([ROLE_ADMIN, ROLE_EDITOR], user)) {
+    return true;
+  }
+
+  // Everyone else can only manage their own content
+  return {
+    createdBy: {
+      equals: user.id,
```

Comment:
@kelvinkipruto this throws an error when run on an initially empty database
![image](https://github.com/user-attachments/assets/47d5dd9e-4d6f-4042-9f27-f550219cee97)